packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -73,7 +73,7 @@
 
 
 .block-editor-global-styles-background-panel__inspector-media-replace-container {
-	border: 1px solid $gray-300;
+	border: $border-width solid $gray-300;
 	border-radius: 2px;
```

Comment:
The rabbit hole is endless. Should these 2px be `$radius-block-ui;`? Or even the new scale, small?

Reply:
Next steps for radius is a bit tricky, we need to:

1. Eliminate any hard-coded values like this
2. _Apply_ the radius scale in the editor
3. Tidy up any unnecessary `border-radius` overrides (there are quite a few in the editor)

So potential next steps would be:

* Replace any hard-coded values like this one with `$radius-block-ui`
* Use `$radius-block-ui` as a to-do list; manually go through and replace instances of it with the appropriate values from the new scale.

Reply:
Zero rush. Doesn't have to be here, or now, or you. Just wanted to mention it as I saw it.